src/NServiceBus.RabbitMQ/Configuration/ConnectionConfiguration.cs

```diff
@@ -31,7 +31,7 @@ class ConnectionConfiguration
 
         public string CertPassphrase { get; set; }
 
-        public IDictionary<string, object> ClientProperties { get; } = new Dictionary<string, object>();
+        public Dictionary<string, object> ClientProperties { get; } = new Dictionary<string, object>();
```

Comment:
Thanks, I missed that the class was internal. (I'm used to always seeing explicit `internal`.)

Agreed, concrete type is fine.